v19.2/insert.md

```diff
@@ -439,204 +471,195 @@ IDs:
 
 {% include copy-clipboard.html %}
 ~~~ js
-var async = require('async');
-
-// Require the driver.
-var pg = require('pg');
+var async = require('async')
+var pg = require('pg')
 
-// Connect to the "bank" database.
+// Config to connect to the "movr" database.
 var config = {
-  user: 'root',
-  host: 'localhost',
-  database: 'bank',
-  port: 26257
-};
-
-pg.connect(config, function (err, client, done) {
-  // Closes communication with the database and exits.
-  var finish = function () {
-    done();
-    process.exit();
-  };
-
-  if (err) {
-    console.error('could not connect to cockroachdb', err);
-    finish();
+    user: 'root',
+    host: 'localhost',
+    database: 'movr',
+    port: 26257
   }
+
+// Create pool
+var pool = new pg.Pool(config)
+
+pool.connect(function (err, client, done) {
+
+    // Close communication with the database and exit.
+    var finish = function () {
+        done()
+        process.exit()
+    }
+
+    if (err) {
+        console.error('could not connect to cockroachdb', err)
+        finish()
+    }
   async.waterfall([
     function (next) {
-      // Insert two rows into the "accounts" table
+      // Insert two rows into the "drivers" table
       // and return the "id" values generated server-side.
       client.query(
-        `INSERT INTO accounts (id, balance)
-         VALUES (DEFAULT, 1000), (DEFAULT, 250)
-         RETURNING id;`,
+        `INSERT INTO drivers (id, city)
+         VALUES (DEFAULT, 'seattle'), (DEFAULT, 'chicago')
+         RETURNING id`,
         next
-      );
+      )
     }
   ],
   function (err, results) {
     if (err) {
-      console.error('error inserting into and selecting from accounts', err);
-      finish();
+      console.error('error inserting into and selecting from drivers', err)
+      finish()
     }
     // Print out the returned values.
-    console.log('IDs:');
+    console.log('IDs:')
     results.rows.forEach(function (row) {
-      console.log(row);
-    });
+      console.log(row)
+    })
 
-    finish();
-  });
-});
+    finish()
+  })
+})
 ~~~
 
 The printed values would look like:
 
 ~~~
 IDs:
-{ id: '190019066706952193' }
-{ id: '190019066706984961' }
+{ id: 'cdd379e3-2d0b-4622-8ba8-4f0a1edfbc8e' }
+{ id: '4224b360-b1b0-4e4d-aba2-a35c64cdf404' }
```

Comment:
All of these code sample updates LGTM at first blush.  After I get through the initial prose/doc review, I'll take a second pass through where I check all the SQL statements and code snippets.  Gosh we need automated testing :-)

Reply:
Sounds good! +1